src/accessors/IConfigurationExtend.ts

```diff
@@ -6,6 +8,12 @@ import { ISlashCommandsExtend } from './ISlashCommandsExtend';
  * of your Rocketlet. It is provided during initialization of your Rocketlet
  */
 export interface IConfigurationExtend {
+    /** Accessor for registering the assets which need to be. */
+    readonly assets: IAssetExtend;
+
+    /** Accessor for declaring the contextual bar buttons your Rocketlet provides. */
+    readonly contextualBarButtons: IContextualBarButtonsExtend;
```

Comment:
If it's going to be used by other technologies than a web browser, which I hope we eventually will allow, then yes I believe it should be.